utils/nctl/sh/scenarios/network_soundness.py

```diff
@@ -54,12 +60,20 @@ def log(msg):
     return
 
 
-def invoke(command):
+def invoke(command, quiet=False):
+    if not quiet:
+        log("invoking command: {}".format(command))
     invoke_lock.acquire()
-    result = subprocess.check_output(['/bin/bash', '-i', '-c',
-                                      command]).decode("utf-8").rstrip()
-    invoke_lock.release()
-    return result
+    try:
+        result = subprocess.check_output(['/bin/bash', '-i', '-c',
+                                          command]).decode("utf-8").rstrip()
+        return result
+    except subprocess.CalledProcessError as e:
+        log("command returned non-zero code - that is expected since the node is temporarily down"
```

Comment:
nit: maybe re-phrase this log message.
```suggestion
        log("command returned non-zero exit code - this can be a transitory error if the node is temporarily down"
```

Reply:
Updated in feffb3b37f8311fb6a1ead1f586a48632d8322b8